add-to-pydotorg.py

```diff
@@ -34,6 +34,8 @@
 
 import requests
 
+from release import run_cmd
```

Comment:
Hummm, this needs to be copied here because this script live standalone in the server. The `release.py` file will not be available there.

Reply:
Checkout https://github.com/python/release-tools/blob/3562448d6042a48d3bf19c7c808654c2137fd4a0/run_release.py#L720-L744

Reply:
Apologies for the trouble 😅

Reply:
I had a suspicion this was the case, but missed that. Should be good now!